test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -42,9 +42,9 @@ const snapshotGroup = "snapshot.storage.k8s.io"
 const snapshotAPIVersion = "snapshot.storage.k8s.io/v1beta1"
 
 var (
-	snapshotGVR        = schema.GroupVersionResource{Group: snapshotGroup, Version: "v1beta1", Resource: "volumesnapshots"}
-	snapshotClassGVR   = schema.GroupVersionResource{Group: snapshotGroup, Version: "v1beta1", Resource: "volumesnapshotclasses"}
-	snapshotContentGVR = schema.GroupVersionResource{Group: snapshotGroup, Version: "v1beta1", Resource: "volumesnapshotcontents"}
+	SnapshotGVR        = schema.GroupVersionResource{Group: snapshotGroup, Version: "v1beta1", Resource: "volumesnapshots"}
```

Comment:
Exported symbols must be documented (https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/88669/pull-kubernetes-verify/1233549485305499653).

Reply:
Thank you for pointing it out. Fixed.